packages/common/src/EncryptionHelper.js

```diff
@@ -18,6 +18,19 @@ function computeVoteHash(request) {
     { t: "int", v: request.salt },
     { t: "address", v: request.account },
     { t: "uint", v: request.time },
+    { t: "bytes", v: "0x" },
```

Comment:
Do the following hashes match?

```
sha3(uint(5), uint(6));
sha3(uint(5), byes(0), uint(6));
```

If _not_, we need to be careful about releasing the common package as it will likely break compatibility with the current voting contract, right?

Reply:
Update: they _do_ seem to match (although someone else should verify :)). Very cool. In that case, is the "0x" element here basically just documentation to show that we're not leaving the value out?

Reply:
yup, it shows we are leaving it out. it also mimics _exactly_ how the hash is generated on-chain in `revealVote(identifier, time, price, "", salt);` with the blank string for the bytes type.

Reply:
It's really really nice that a 0 length data is perfectly backwards compatible when doing tight packing 👌